packages/webpack/CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+# Changelog
+
+## [0.0.1-beta.0](https://github.com/LavaMoat/LavaMoat/compare/webpack-v0.0.1-beta.0...webpack-v0.0.1-beta.0) (2023-10-18)
```

Comment:
I think we should remove this later when we're out of beta. The mess below is actually useful now and I'm happy with how it's generated despite some artifacts from when it was still called scorchwrap